src/fruity/plist.vala

```diff
@@ -1,36 +1,37 @@
 namespace Frida.Fruity {
 	public class Plist : PlistDict {
-		public const uint8 FMT_AUTO = 0;
-		public const uint8 FMT_BIN = 1;
-		public const uint8 FMT_XML = 2;
+		public enum Format {
+			AUTO,
+			BINARY,
+			XML
+		}

```

Comment:
```suggestion
		}

```

Reply:
newline added to line 8 in 1d1de403be669436eeb3eaa0cb79ac62a9e89042